src/Algebra/Bundles.agda

```diff
@@ -781,19 +781,19 @@ record IdempotentSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
     )
 
 record KleeneAlgebra c ℓ : Set (suc (c ⊔ ℓ)) where
-  infix  8 _⁻*
-  infixl 7 _*_
+  infix  8 _*
+  infixl 7 _∙_
```

Comment:
Hmm I'm not sure like this change. The symbol `*` is used as multiplication in all the other algebraic structures, so I think it's pretty confusing to both a) renaming it to `∙` and then b) use `*` for something else.

Did you have a problem with my earlier suggestion of using the actual `\star` for star?

Reply:
My bad sorry I will change it soon

Reply:
As @WolframKahl suggested it doesn't look neat `* ⋆`. Should I still change it?

Reply:
Well we do need to stay consistent with the rest of the library so I'm going to insist on turning multiplication back to `*`. If you have a better idea about what to name the star operation then I'm open to alternative suggestions :smile: